tensorflow/compiler/tf2tensorrt/kernels/trt_engine_op.cc

```diff
@@ -1050,7 +1059,8 @@ StatusOr<std::pair<EngineContext*, int>> TRTEngineOp::GetEngine(
     const std::vector<TensorShape>& input_concrete_shapes, OpKernelContext* ctx,
     TRTEngineCacheResource* cache_res) {
   static EngineContext empty_context;
-
+  tensorflow::profiler::TraceMe activity(
```

Comment:
this is not your fault, but the merge process starts to complain the existing code. There are 9 place where absl:: is used but it is now required to use std::.  Here are the line numbers
Line 1052
Line 1114
Line 1132
Line 1164
Line 1206
Line 1224
Line 1240
Line 1339
Line 1345

Reply:
Oh sure, let me fix that :)

Reply:
PR: #56625